src/pages/components/select/usage.mdx

```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
+
+### States
+
+The select has multiple interactive states for both default and inline variants:
+**enabled**, **hover**, **focus**, **open**, **error**, **warning**, and
+**disabled**.
+
+| State       | When to use                                                                                                                                                                                                                                                  |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| _Enabled_   | When the select is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component. An enabled select field should contain a default value.                                                |
+| _Hover_     | When a user's mouse cursor is hovering over the field.                                                                                                                                                                                                       |
+| _Focus_     | When a user tabs to or clicks on the select field, the field becomes focused, indicating the user has successfully navigated to the component.                                                                                                               |
+| _Error_     | When the required select value has not been selected. It can also be triggered due to a system error. This state requires a user response before data can be submitted or saved.                                                                             |
+| _Warning_   | When you need to call the user's attention to an exception condition. The condition might not be an error but can cause problems if not resolved.                                                                                                            |
+| _Disabled_  | When the user is not allowed to interact with the select due to either permissions, dependencies, or prerequisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+| _Skeleton_  | Used on an initial page load to indicate that the number input has not yet fully loaded.                                                                                                                                                                     |
+| _Read-only_ | _Coming soon!_                                                                                                                                                                                                                                               |
+
+<Row>
+<Column colLg={12}>
+
+![Select interactive states](images/select-usage-9.png)
```

Comment:
— I think you could remove the word `state` in the labels of each component in the image.
— The disabled state does not get a bottom-border rule.